src/elements/content-preview/ContentPreview.js

```diff
@@ -1323,7 +1323,12 @@ class ContentPreview extends React.PureComponent<Props, State> {
                             canDownload={this.canDownload()}
                             canPrint={canPrint}
                             onDownload={this.download}
-                            contentAnswersProps={contentAnswersProps}
+                            contentAnswersProps={{
+                                ...contentAnswersProps,
+                                fileId: currentFileId,
+                                fileExtension: currentExtension,
+                                versionId: selectedVersionId,
```

Comment:
these are also set in `PreviewHeader`:
https://github.com/box/box-ui-elements/blob/c987daea3de9e44eb54f363641750215bbf08613/src/elements/content-preview/preview-header/PreviewHeader.js#L66-L68

so no need to pass from this component

Reply:
I literally removed this before you made the comment